development/image-syncer/external-images.yaml

```
@@ -34,8 +34,8 @@ images:
 - source: "oryd/oathkeeper-maester:v0.1.5"
   amd64Only: true
 - source: "oryd/oathkeeper:v0.38.25-beta.1"
-- source: "postgres@sha256:45800340cfea631af4c8c3870e8d7f1ae888907e39def41a3b7608ff8f16b28c"
-  tag: "11.20-alpine3.18"
+- source: "postgres@sha256:66c2a66fb183d40cde6ace5e292e2d92abc07eb447873a1a64fdb729f7b7c8cf"
```

as per `docker run mplatform/manifest-tool inspect postgres:11.21-alpine3.18` mutiplatform SHA is different
```suggestion
- source: "postgres@sha256:86c55091a74b9efe95396f65ddd83f0067ff47c3f09993621d6f94c835856445cf"
```